src/i18n/en/translation.json

```diff
@@ -29,7 +29,9 @@
       "column_one": "{{count}} Column",
       "column_other": "{{count}} Columns",
       "start": "Start",
-      "save": "Save"
+      "save": "Save",
+      "signInToCreateBoards": "Sign in to create a board.",
+      "anonymousCannotCreateBoards": "Anonymous users cannot create new boards"
```

Comment:
I think the translated text could be a little more concise like the text above, since anonymous users are also logged in, just anonymous. Maybe something like "the admin has disabled boards creation for anonymous users"?